api/processors/auxil/log_event_creator.py

```diff
@@ -7,14 +9,20 @@
 )
 
 
+class FromOrToGroup(enum.Enum):
+    FROM = "from_group"
+    TO = "to_group"
```

Comment:
There is also a possibility that a student or a teacher is transferred from one group to another. Does the general design of your solution support this case?

Also, I think I understand why it's in this module (as it's only related to group events), maybe that's why you didn't declare this class somewhere in e.g. `enums.py`. Just checking.

Reply:
Redesigned it to use both from_group and to_group, so removed the enum